andela/ah-frontend-space
# #166880069 Update travisCI to run tests.

#### What does this PR do?

Update travisCI to run tests

#### Description of Task to be completed?

- enable github repository to run tests on travisCI
- add .travis.yml file for configurations

#### How should this be manually tested?
Go to this branch on github
There should be a badge showing passing tests
click on the bagde to view the log on traviCI

### Screenshots.
<img width="1006" alt="trav" src="https://user-images.githubusercontent.com/40593659/60754594-f8b12280-9feb-11e9-8553-5e3880761a64.png">


#### What are the relevant pivotal tracker stories?

[#166880069](https://www.pivotaltracker.com/story/show/166880069)
